tools/build.py

```diff
@@ -219,6 +229,8 @@ def adjust_options(options):
     # Then add calculated options
     options.host_tuple = '%s-%s' % (platform.arch(), platform.os())
     options.target_tuple = '%s-%s' % (options.target_arch, options.target_os)
+    options.simple_target_tuple = '%s-%s' % (arch_name(options.target_arch),
+        os_name(options.target_os))
```

Comment:
function name like `os_name` and `arch_name` can be misleading.  How about directly changing like others above?
```python
if options.target_arch == 'x86':
  options.target_arch = 'i686'
if options.target_arch == 'x64':
  options.target_arch = 'x86_64'
```